ui/app/mainui/ToastsManager.qml

```diff
@@ -169,6 +202,22 @@ QtObject {
         case ToastsManager.ActionType.OpenSendModalPopup:
             root.sendModalPopup.open()
             return
+        case ToastsManager.ActionType.ViewTransactionDetails:
+            var txHash = ""
+            if(actionData) {
+                var parsedData = JSON.parse(actionData)
+                txHash = parsedData.txHash
+                Global.changeAppSectionBySectionType(Constants.appSection.wallet,
```

Comment:
I wonder if this will work correctly; AppMain.qml has this handler code:
```qml
if (sectionType === Constants.appSection.wallet) {
    appView.children[Constants.appViewStackIndex.wallet].item.openDesiredView(
                WalletLayout.LeftPanelSelection.AllAddresses,
                WalletLayout.RightPanelSelection.Activity,
                data
                )
}
```

Reply:
It does work (video 0.34s). It's the same mechanism used in other parts of the app like in `SavedAddressesDelegate`. Indeed, if I'm not wrong it's the code that does the magic, navigating deeply to the "subsection", no?

Reply:
Yeah hopefully... just didn't see how it could work at the first glance :)